### PR TITLE
1 task

### PR DESCRIPTION
**Pull request by Levii01**
Installed devise and created a section with users listing. Created simple page design with bootstrap and partial for messages.
@socjopata I hope that I didn't make any mistake in creating a new repository. In next commit i will update README.md file.

### CODE REVIEW

**Comment by Levii01** [comment]
@socjopata Can I merge ?

File: app/views/layouts/_messages.html.erb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<% flash.each do | name, msg |%>
+  <div class="alert alert-<%= select_class_alert(name)%>">
```
Comment (socjopata) [inline app/views/layouts/_messages.html.erb]: Consider being consistent, as here you're iterating with `name` and in the method definition you operate with `type`

File: spec/helpers/application_helper_spec.rb [hunk 2]
```diff
@@ -0,0 +1,16 @@
+require 'rails_helper'
+
+RSpec.describe ApplicationHelper, type: :helper do
+  describe "#which_alert" do
+    let(:success) { 'notice'}
+    let(:danger) { 'danger'}
+
+    it 'return success alert' do
```
Comment (socjopata) [inline spec/helpers/application_helper_spec.rb]: return -> returns.

File: spec/models/user_spec.rb [hunk 3]
```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  require File.expand_path(File.dirname(__FILE__) + '/../spec_helper')
+
+  describe 'create new user' do
+    context 'when create new user' do
```
Comment (socjopata) [inline spec/models/user_spec.rb]: when create -> when creating

**Review by socjopata — APPROVED** [review]
Consider implementing the feedback. You have a green light to merge

File: app/views/layouts/_messages.html.erb (hunk 1 above)
Comment (Levii01) [reply app/views/layouts/_messages.html.erb]: My wrong, I changed it :)

**Pull request merged**